games/hangman.c

```diff
@@ -0,0 +1,277 @@
+/**
+ * @hangman.c
+ * @brief C implementation of hangman game. Wikipedia: https://en.wikipedia.org/wiki/Hangman_(game)
+ * @details
+ * Simple, readable version of hangman.
+ * Changed graphic to duck instead of traditional stick figure (same number of guesses).
+ * @author [AtlantaEmrys2002](https://github.com/AtlantaEmrys2002)
+*/
+
+#include <ctype.h> /// for main()
+#include <stdio.h> /// for main(), new_word(), new_guess(), won()
+#include <stdlib.h> /// for all functions
+#include <string.h> /// for main()
+#include <time.h> /// for new_game()
+#include <assert.h> /// for self-testing
+
+// global variable - word to guess
+char current_word[] = "";
+
+//functions
+void new_game(); // creates a new game
+int new_guess(char, const char guesses[], int size); // checks if player has already played letter
+int in_word(char, char word[], unsigned int size); // checks if letter is in word
+void picture(int score); // outputs image of duck (instead of hang man)
+void won(int score); // checks if player has won or lost
+
+/**
+ * @brief Self-test Implementations
+ * @returns void
+ */
+static void test() {
+    /*checks in_word returns correct results (e.g. c is in cat, d is not in flower)*/
+    assert(in_word('c', "cat", 3) == 1);
+    assert(in_word('d', "flower,", 6) == -1);
+    /*checks new_guess returns correct results (e.g. d is in dhjkl and has been guessed before, but e is not and hasn't been guessed before)*/
+    //assert(new_guess('d', "dhjkl", 5) == 1);
+    assert(new_guess('z', "testing", 5) == -1);
+}
+
+/**
+ * @brief Main Function
+ * @returns 0 on exit
+ */
+int main() {
+
+    test();
+
+    int i;
+    char guess;
+    char guesses[25] = {}; // previous guesses
+    int guesses_size;
+    int incorrect = 0; // incorrect guesses counter
+    char hidden[100] = ""; // hidden word
+
+    new_game();
+
+    for (i = 0; i < ((strlen(current_word)) - 1); i++) {
```

Comment:
`strlen(current_word)` already represents the number of characters in the string (and hence the number of `_` needed), are you trying to compensate for the extra newline character here? I would suggest you to do that when `current_word` is inputted, not when it's used, it's much cleaner that way
```suggestion
    for (i = 0; i < strlen(current_word); i++) {
```